HealthcareApp/src/Healthcare.Application/Mappings/MappingProfile.cs

```diff
@@ -0,0 +1,22 @@
+using Application.DTOs.Register;
+using AutoMapper;
+using Domain.Constants;
+using Domain.Entities;
+using Healthcare.Application.Appointments.Commands.Book;
+using static Domain.Constants.AppointmentStatuses;
+
+
+namespace Healthcare.Application.Mappings;
+
+public class MappingProfile : Profile
+{
+    public MappingProfile()
+    {
+        CreateMap<BookAppointmentCommand, Appointment>()
+            .ForMember(dest => dest.EndTime, opt => opt.MapFrom(src => src.StartTime.Add(TimeSpan.FromMinutes(15))))
```

Comment:
Never pass any business logic to mapping. Mapping process is only **projecting 1 property to another**.
Noone will expect any logic here and when they see that after mapping their endtime is startTime + 15min they will be confused